### PR TITLE
Update events, and improve styling

### PR DESCRIPTION
**Pull request by mirisuzanne**
## Cute animal pic
_Because everyone likes pictures of [animals](https://unsplash.com/t/animals)._

Grumpy Cat, but not GrumpyCat

## Trello Card

https://trello.com/c/lYbTj6Kd/110-event-updates-chore

## Description
_The commit messages say what you did; this should explain why and/or how._

- Update Miriam's upcoming events
- Update slide/video data for past events
- Improve event filters/macros and styles
- Improve filter docs

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

- `/authors/miriam/`
- `/talks/css-is-rad/`
- `/sample/filters/events/`

## Show me
_Provide screenshots/animated gifs/videos if necessary._

Author Page:
screenshot

Talk Detail Page:
screenshot

### CODE REVIEW

**Comment by mirisuzanne** [comment]
[Event updates [chore]](https://trello.com/c/lYbTj6Kd/110-event-updates-chore)

File: content/_includes/events.macros.njk [hunk 1]
```diff
@@ -38,16 +39,50 @@ params:
...
       {%- for event in events -%}
-        <li>
-          {%- if event | isFuture -%}
-            <strong>[{{ utility.datetime(event.date) }}]</strong>
-          {%- else -%}
-            [{{ utility.datetime(event.date) }}]
-          {%- endif %}
-          {{ utility.link_if(event.venue, event.url) }}{% if event.adr %} {{ '--' | typogr | safe }}
-          {{ event.adr }}{% endif %}
+        {% set event_slug = [event.date, loop.index] | join('-') | slug %}
```
Comment (jgerigmeyer) [inline content/_includes/events.macros.njk:44]: This creates really long/weird IDs: `id="mon-oct-26-2020-00:00:00-gmt+0000-(coordinated-universal-time)-1"`

**Review by jgerigmeyer — APPROVED** [review]

**Review by jgerigmeyer — APPROVED** [review]

**Review by SondraE — APPROVED** [review]
I couldn't find any issues.

**Review by dvdherron — APPROVED** [review]
Looks good to me.

**Pull request merged**